tests/runtime/test_service_factory.py

```diff
@@ -63,6 +63,18 @@ def compiled_training_service() -> ServicePackage:
     )
 
 
+def test_service_package_raises_for_compiled_training_management():
+    with pytest.raises(CaikitRuntimeException) as e:
+        ServicePackageFactory.get_service_package(
+            ServicePackageFactory.ServiceType.TRAINING_MANAGEMENT,
+            ServicePackageFactory.ServiceSource.COMPILED,
+        )
+    assert (
```

Comment:
something cool I found the other day is that `pytest.raises` has a `match` keyword that takes regexes to match on the exception message so that you don't have to do this!

```
with pytest.raises(CaikitRuntimeException, match="Not allowed to get Training Management services from compiled packages"):
```